README.md

```diff
@@ -1 +1,190 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+<a href="https://github.com/JavierAybar/hello-react-front-end/tree/setup-front-end">See the React Application</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+    <li><a href="https://tailwindcss.com/">TailwindCSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [x] The user logs in to the website by typing the Email and Password.
+- [x] In the navigation panel, the user can see links to:
+- "Items list".
+- "Reserve" form.
+- "My reservations".
+- "Add items".
+- "Delete items" (the link is visible to everybody).
+- [x] On the main page, the user can see a list of motorcycles/doctors/classes/items that you selected as a theme.
+List
+
+- [x] When the user selects a specific item, they can see the details page with its full description.
+- [x] On the details page, the user can click the "Reserve" button.
+- [x] When the user clicks the "Add item" link in the navigation panel they can see a form for adding a new item.
+- [x] Make the app responsive, creating both mobile and desktop versions.
+
+- [x] When the user clicks the "Delete item" link in the navigation panel they can see a list of all items with title and "Delete" button.
+- [x] When the user clicks the "Delete" button, the selected item is marked as removed and does not show on the main list anymore.
+- [x] To reserve an appointment, the user has to select a date and city (username and selected item are auto-filled).
+- [x] Have all necessary inputs.
+- [x] The user can access the "Reserve" page from the navigation panel. In that case, only the username is auto-filled.
+
+- [x] When the user clicks the "My reservations" link in the navigation panel they can see a list of their reservations (with information about item name, date, and city).
+- [x] Have full documentation for the API.
+- [x] Implement proper user authentication from the front end to the server.
+- [x] The "Add item" and "Delete item" links are accessible only by users who are admins.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Install <a name="install"></a>
```

Comment:
- [ ] When I try to setup the database, I get this:

![image](https://github.com/JavierAybar/Full_Stack_Capstone_Back_End/assets/57421823/d9340b21-16c9-49fa-a301-e0fa5e04c79a)

This is because your users will need to create their own master.key.  These are the instructions:

```
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. Close the editor that opens.
4. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

Result:

![image](https://github.com/JavierAybar/Full_Stack_Capstone_Back_End/assets/57421823/cf23f100-04a7-4917-a574-d65c37e5561a)


Kindly update your instructions to include how to create the master.key so your users are able to setup your project correctly. 👍